packages/shared/src/hooks/useFeed.ts

```diff
@@ -166,8 +167,13 @@ export default function useFeed<T>(
             index,
           }));
 
-          if (settings.showAcquisitionForm) {
+          if (pageIndex === 0 && settings.showAcquisitionForm) {
             posts.splice(adSpot, 0, { type: 'userAcquisition' });
+          } else if (pageIndex === 0 && !!settings.marketingCta) {
```

Comment:
So we're only adding this if the acquisition form is not shown?

Reply:
It's maybe a good product question.
Since they take the same placement, not sure which takes priority.

Assumed UA had higher prio since it's only on onboarding?
@omBratteng can you raise it for product?

Reply:
Asked product.

Reply:
> CTA since people get “get stuck” on the acquisition card

So CTA takes priority.